.github/workflows/testing.yml

```diff
@@ -231,3 +231,78 @@ jobs:
           python -m pip install --upgrade .
       - name: Test PDF generation on Windows
         run: pytest test/test_output_engine.py -k test_output_pdf
+
+  windows_long_tests:
+    name: Windows long tests
+    runs-on: windows-2019
```

Comment:
Is there a reason we need `windows-2019` instead of `windows-latest`? (I'm guessing you left it as `windows-2019` while debugging the checker issue)

Reply:
Yeah I was using it while debugging